docs/en/siem/detection-engine-intro.asciidoc

```diff
@@ -77,25 +77,50 @@ To investigate a signal in the Timeline, click the *View in timeline* icon.
 [[detections-permissions]]
 === {kib} and index privileges required for Detections
 
-[IMPORTANT]
+Depending on your privileges and whether a `.siem-signals-<space name>` index 
+has been created for the {kib} space, you might see an error message when you 
+try to open the *Detections* page.
+
+[NOTE]
 ==============
-You cannot view the Detections page until a user with the `create_index` 
-privilege for the {kib} space visits the page.
+If you are using a local {es} deployment and cannot access the {siem-app}, make 
```

Comment:
> and cannot access the {siem-app}

It is possible run a local (non-Elastic Cloud) instance without security enabled and still access the SIEM app, however the user will see the `Detection engine permissions` required message on the Overview page (in place of the `Signals` histogram), and the same message will appear in the Detections page.

Consider changing:
```
and cannot access the {siem-app}
```

to something like:
```
and you see the 'Detection engine permissions' required message in the Overview or Detections page
```